hack/builder/publish.sh

```diff
@@ -29,5 +29,9 @@ for ARCH in ${ARCHITECTURES}; do
 done
 
 export DOCKER_CLI_EXPERIMENTAL=enabled
+# The below commands are not working with podman container runtime.
+# Using docker container runtime to create and push the manifest.
+
+export KUBEVIRT_CRI=docker
```

Comment:
Where does this line come from and is it needed?

Reply:
@jschintag  The command to create and push the manifest are not working with any other container runtimes execpt docker. So i am setting the KUBEVIRT_CRI to docker

Reply:
@vamsikrishna-siddu - which version of podman did you try this with? It should be working on some of the later versions of podman

Reply:
@brianmcarey 
```
Version:      3.4.2
API Version:  3.4.2
Go Version:   go1.15.2
Built:        Thu Jan  1 00:00:00 1970
OS/Arch:      linux/amd64
```

Reply:
I think it works in podman version > 4.4 

Anyway this script is used by a CI job to publish the builder images so updates here should not be required.

Reply:
Sure @brianmcarey  I verified in 4.4.1 version it is working. Let me remove it. Thanks